compose.yml

```diff
@@ -0,0 +1,35 @@
+version: "3.5"
+
+networks:
+  sliding-sync:
+
+services:
+  slidingsync:
+    image: ghcr.io/matrix-org/sliding-sync
+    container_name: slidingsync
+    environment:
+      - SYNCV3_SERVER=https://matrix-client.matrix.org
+      - SYNCV3_SECRET=  #use `openssl rand -hex 32` to generate a secret to paste here
```

Comment:
I'm actually using this in production, but I guess it can suit well for development too...
I'd leave it up to you to if you as project maintainer if you want to include a pre-generated secret for a dev-setup that would have to be changed for production use or leave the field empty.
for defaulting to matrix.org: I did this as the Readme defaults to matrix.org as well (docker command in Readme section - that's what I've used as a base to build this compose.yml)... I personally use this with my own homeserver to test-drive Element X... let me know if you want me to remove the default :)